index.html

```diff
@@ -5,45 +5,364 @@
+{"num":19,"neighborhood":"Lamond Riggs","poverty_rate":14,"avg_family_income":75912},
+{"num":14,"neighborhood":"Glover Park","poverty_rate":7.9,"avg_family_income":157349},
+{"num":17,"neighborhood":"Takoma","poverty_rate":13,"avg_family_income":81956},
+{"num":16,"neighborhood":"Shepherd Park","poverty_rate":3.8,"avg_family_income":180800},
+{"num":36,"neighborhood":"Garfield Heights","poverty_rate":45,"avg_family_income":45220},
+{"num":31,"neighborhood":"Deanwood","poverty_rate":34,"avg_family_income":51670},
+{"num":32,"neighborhood":"Fort DuPont","poverty_rate":24,"avg_family_income":57825},
+{"num":12,"neighborhood":"Van Ness","poverty_rate":8.7,"avg_family_income":217847},
+{"num":30,"neighborhood":"Mayfair","poverty_rate":32,"avg_family_income":37659},
+{"num":24,"neighborhood":"Woodridge","poverty_rate":13,"avg_family_income":97305},
+{"num":29,"neighborhood":"Kenilworth","poverty_rate":41,"avg_family_income":42821},
+{"num":18,"neighborhood":"Petworth","poverty_rate":14,"avg_family_income":88437},
+{"num":25,"neighborhood":"Kingman Park","poverty_rate":14,"avg_family_income":129089},
+{"num":28,"neighborhood":"Anacostia","poverty_rate":44,"avg_family_income":34195},
+{"num":34,"neighborhood":"Fairlawn","poverty_rate":21,"avg_family_income":70274},
+{"num":38,"neighborhood":"Shipley Terrace","poverty_rate":38,"avg_family_income":38427},
+{"num":33,"neighborhood":"Capitol View","poverty_rate":22,"avg_family_income":53239},
+{"num":23,"neighborhood":"Ivy City","poverty_rate":32,"avg_family_income":48388},
+{"num":39,"neighborhood":"Bellevue","poverty_rate":36,"avg_family_income":41605},
+{"num":9,"neighborhood":"Southwest","poverty_rate":19,"avg_family_income":94339},
+{"num":22,"neighborhood":"Brookland","poverty_rate":23,"avg_family_income":81931},
+{"num":1,"neighborhood":"Adams Morgan","poverty_rate":5.6,"avg_family_income":162891},
+{"num":37,"neighborhood":"Barry Farm","poverty_rate":41,"avg_family_income":38015},
+{"num":8,"neighborhood":"Downtown","poverty_rate":26,"avg_family_income":142881},
+{"num":35,"neighborhood":"Naylor Gardens","poverty_rate":17,"avg_family_income":72747},
+{"num":6,"neighborhood":"DuPont Circle","poverty_rate":13,"avg_family_income":177893},
+{"num":20,"neighborhood":"Michigan Park","poverty_rate":11,"avg_family_income":87062},
+{"num":21,"neighborhood":"Edgewood","poverty_rate":17,"avg_family_income":91520},
+{"num":2,"neighborhood":"Columbia Heights","poverty_rate":17,"avg_family_income":87204},
+{"num":7,"neighborhood":"Logan Circle","poverty_rate":12,"avg_family_income":123697},
+{"num":3,"neighborhood":"Shaw","poverty_rate":17,"avg_family_income":101583},
+{"num":27,"neighborhood":"Navy Yard","poverty_rate":34,"avg_family_income":77952}];
+
+var MAX_INCOME = 304995;
+var MAX_POVERTY = 45;
+
+var options = {
+  sorted: false,
+  filtered: false,
+  page:0
+};
+
+
+
+
+function getNumberOfPages() {
+  return Math.ceil(dc_info.length/numberPerPage);
+}
+
+function nextPage() {
+  currentPage += 1;
```

Comment:
I'm assuming this is just stray code, as it's no longer being called. But note that this is an "implicit global variable", and it's frowned on. You may have reached this point because you had `var currentPage` here, and realized that it wasn't sticking around between calls of `nextPage()`. The solution, though, is just to declare `var currentPage` outside of the scope (the {}) of nextPage. Like this:

```js
var currentPage = 0;
function nextPage() { return currentPage += 1; }
```